src/cfn-resource-mappings.ts

```diff
@@ -26,90 +26,13 @@ export function mapToCfnResource(
     rawProps: any,
     options: pulumi.ResourceOptions,
 ): ResourceMapping {
-    const props = normalize(rawProps);
+    const props = normalize(rawProps, typeName);
     debug(`mapToCfnResource typeName: ${typeName} props: ${JSON.stringify(props)}`);
     switch (typeName) {
-        case 'AWS::AppRunner::Service':
```

Comment:
Quick comment on why this all is going away?

Reply:
All of these are either:

- legacy mappings when we were not correctly auto mapping these resources
- mappings due to JSON types

There should be tests covering each of these to ensure nothing is actually changing.